src/Redeemer.sol

```diff
@@ -649,6 +649,19 @@ contract Redeemer {
         return incentiveFee;
     }
 
+    /// @notice Allows the admin to externally depsoit underlying for a market
+    /// @notice This is to be used in emergency situations where the redeem method is not functioning for a market
+    /// @param u address of the underlying asset
+    /// @param m maturity of the market
+    /// @param a amount of underlying to be deposited
+    function depositHoldings(address u, uint256 m, uint256 a) external authorized(admin) {
```

Comment:
I dont see a particular reason to restrict the method to admins.

Though yea we should be the only ones that use it, im dreaming of some wild edge case where an external address might deposit or make some renumeration.